support-frontend/assets/helpers/internationalisation/country.js

```diff
@@ -421,20 +425,26 @@ export const isInStripePaymentRequestAllowedCountries = (country: IsoCountry) =>
   stripePaymentRequestAllowedCountries.includes(country);
 
 // ----- Functions ----- /
+function sateFromMap(l: string, states: {[p: string]: string}): ?StateProvince {
+  const s = l.toUpperCase();
+  return states[s] ? s :
+    Object.keys(states).find(key => states[key].toUpperCase() === s || (s.length === 3 && s.startsWith(key)));
```

Comment:
Should we call `toUpperCase()` on `s` as well, so we know we are comparing like for like?